### DISCUSSION TITLE
게임 비쥬얼

### DISCUSSION THREAD

**Discussion by mille-feu** [Tiny-Metro/Poly-Metro#16] [Ideas]
2.5D로 구성된 만큼, 원본 게임과 같이 완전 평면적으로 구성할 경우 어색할 수 있습니다.
- 역의 모델
- 맵의 구성 요소 (평지, 강, 언덕)
- 기타

의견 바랍니다.

**Comment by cstrnull00** [comment]
- 역 모델
  - 평면 도형
- 맵의 구성 요소
  - 강, 언덕, 기차, 다리, 터널 등의 오브젝트는 로우 폴리곤 모델로 렌더링
  - 다음 이미지 참고 (PolyBridge)
  ![image](https://user-images.githubusercontent.com/46642837/202708547-cdccab65-83f1-4570-bc21-42d0af11f315.png)
- 기타 : 맵에서 카메라의 각도 및 기능
  - zoom in, zoom out 시 카메라 각도가 낮아짐
  - 현재 테두리에서 커서를 바깥에 두면 카메라 이동 (모바일에서는 간편하게 드래그로 이동)
  - 오른쪽 마우스를 클릭하고 돌리면 조망 각도 조정 가능, 그러나 오른쪽 마우스 놓을 시 원래대로 이동
  - 다음 영상 참고
![Victoria_3_2022-11-18_20-22-07_AdobeExpress-min](https://user-images.githubusercontent.com/46642837/202696210-956fc9c7-bfd2-4a44-bf48-08f608bc976f.gif)
